board/safety/safety_honda.h

```diff
@@ -6,6 +6,13 @@
 //      accel rising edge
 //      brake rising edge
 //      brake > 0mph
+
+// TODO: -1 bus check to indicate that bus check is skipped. Temp solution
+AddrCheckStruct HONDA_RX_CHECKS[] = {{{0x1A6, 0x296}, 2U, -1, true, 3U, 40000U, true, 0, 0U, 0U, true},
```

Comment:
I would replace the addresses by defines/constants. The code is now riddled with these magic numbers.

Reply:
I'll clean up in another PR. what's done in safety_volkswagen is better indeed.